pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -0,0 +1,1421 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package hypervisor
```

Comment:
May be a silly question, but I will ask anyways. 😁 

How much of this file - functions and structures - are already in `kubevirt.io/api/core`? Since it already is all self-contained there, I would expect most of these structs and funcs already would be there? Or is this the stuff we have to duplicate to avoid massive dependency sprawl?

Reply:
I am confused with your question. kubevirt.go implements the Hypervisor interface which is defined in hypervisor.go for kubevirt eve, just like kvm.go in kvm based eve. As part of the implementation we will be calling in kubevirt/kubernetes  specific API. What duplication are you seeing ?